var/spack/repos/builtin/packages/orca/package.py

```diff
@@ -50,15 +55,27 @@ class Orca(Package):
     depends_on("libpciaccess", type="run")
 
     # Map Orca version with the required OpenMPI version
-    openmpi_versions = {"4.0.1.2": "2.0.2", "4.2.0": "3.1.4", "4.2.1": "3.1.4", "5.0.3": "4.1.2"}
+    openmpi_versions = {
+        "4.0.1.2": "2.0.2",
+        "4.2.0": "3.1.4",
+        "4.2.1": "3.1.4",
+        "5.0.3": "4.1.1",
```

Comment:
Great catch: `4.1.2` to `4.1.1` (to fix the url).

Reply:
Please leave them 4.1.2, 4.1.1 causes issues in our environment, and I suspect anyone running with openmpi schedulers=slurm